Lets Do This/Views/Campaign/LDTCampaignListCampaignCell.m

```diff
@@ -90,13 +96,29 @@ -(void)setExpanded:(BOOL)expanded {
 		self.titleLabelTopLayoutConstraint.constant = CGRectGetHeight(self.imageView.bounds)-CGRectGetHeight(self.titleLabel.bounds)-10; // -10 for padding
 		self.imageViewTop.constant = kCampaignImageViewConstantExpanded;
 		self.imageViewBottom.constant = kCampaignImageViewConstantExpanded;
-		
+        self.imageViewHeight.constant = 242;
+        self.actionButtonHeight.constant = 50;
+        self.actionButtonTop.constant = 16;
+        self.actionButtonBottom.constant = 16;
+		[self.actionButton enable:YES];
+        self.actionButton.hidden = NO;
+        self.taglineTop.constant = 16;
+        self.taglineLabel.hidden = NO;
+
 		[self layoutIfNeeded];
 	}
 	else {
 		self.imageViewTop.constant = kCampaignImageViewConstantCollapsed;
 		self.imageViewBottom.constant = kCampaignImageViewConstantCollapsed;
+        // Desired height is 150, but need 25 with the imageView constant.
+        self.imageViewHeight.constant = 175;
 		self.titleLabelTopLayoutConstraint.constant = self.collapsedTitleLabelTopLayoutConstraintConstant;
+        self.actionButton.hidden = YES;
+        self.actionButtonHeight.constant = 0;
+        self.actionButtonTop.constant = 0;
+        self.actionButtonBottom.constant = 0;
+        self.taglineTop.constant = 0;
```

Comment:
This follows the approach suggested here, http://stackoverflow.com/a/19586924/1470725, which isn't the nicest. Attempts setting `actionButton` to hidden because the button still displays when expanded.